src/task.rs

```diff
@@ -1436,14 +1477,29 @@ impl TaskMonitor {
     #[cfg(test)]
     pub const DEFAULT_SLOW_POLL_THRESHOLD: Duration = Duration::from_millis(500);
 
+    /// The default duration at which polls cross the threshold into being categorized as 'slow' is
+    /// 50μs.
```

Comment:
This is the default duration at which *scheduling* delays cross the threshold into the 'slow' category — not poll delays.